example/nextjs/pages/_app.js

```diff
@@ -48,7 +49,9 @@ export default class MyApp extends App {
               <button
                 type="button"
                 onClick={() => {
-                  i18n.changeLanguage(i18n.languages[0] === 'en' ? 'de' : 'en');
+                  const lng = i18n.languages[0] === 'en' ? 'de' : 'en'
+                  document.cookie = cookie.serialize('i18next', lng)
```

Comment:
This should be not needed:

You could enable cookie on browser language detector too: https://github.com/i18next/i18next-browser-languageDetector#detector-options

`caches: ['localStorage', 'cookie'],`


add it here: https://github.com/i18next/react-i18next/blob/master/example/nextjs/i18n.js#L28